compose_rl/data/dataloader.py

```diff
@@ -102,7 +105,10 @@ def build_preference_dataloader(
             persistent_workers=persistent_workers,
             timeout=timeout,
         )
-        return dataloader
+        data_spec_kwargs = {}
+        if token_counter_fn is not None:
+            data_spec_kwargs['get_num_tokens_in_batch'] = partial(token_counter_fn, pad_token_id=tokenizer.pad_token_id)
```

Comment:
In theory, we can remove the `pad_token_id` if we just stick with the `action_mask + prompt_len` but was curious to hear what is preferred.

Reply:
I think the `action_mask + prompt_len` would probably be the better route since `sequences` isn't really used outside of the callback whereas `action_mask` is used for training